app/pages/applicantportal/dashboard.tsx

```diff
@@ -77,14 +77,23 @@ const Dashboard = ({
 
   const closeTimestamp = openIntake?.closeTimestamp;
   const isInternalIntakeEnabled = useFeature('internal_intake').value ?? false;
+  const [isApplicationCreated, setIsApplicationCreated] = React.useState(false);
```

Comment:
Total, completely insignificant nit but we don't use the pattern `React.useState` anywhere that I am aware of 😅 Not a huge deal though.

Reply:
Good point, for some reason I remembered that being used. Updated